fastlane/lanes/localization.rb

```diff
@@ -98,12 +98,12 @@
   lane :update_appstore_strings do |options|
     # If no `app:` is specified, call this for both WordPress and Jetpack
     apps = options[:app].nil? ? %i[wordpress jetpack] : Array(options[:app]&.downcase&.to_sym)
+    version = options.fetch(:version, android_get_app_version)
 
     apps.each do |app|
       app_values = APP_SPECIFIC_VALUES[app]
 
       metadata_folder = File.join(PROJECT_ROOT_FOLDER, 'WordPress', app_values[:metadata_dir])
-      version = options.fetch(:version, android_get_app_version)
```

Comment:
The version value is not application specific, in fact it's [fetched](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_version_helper.rb#L46) from [`version.properties` file](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/version.properties) which has a shared value. So, there is no harm in moving this outside of the loop and reuse the value in `release_branch` variable.